recipe/recipe.yaml

```diff
@@ -5,7 +5,7 @@ context:
   version: 1.1.0
 
 recipe:
-  name: ${{ name }}
+  name: ${{ name }}-suite
```

Comment:
For reasons unclear to me, having this name is causing it to show up in the README output, when it should just be the names of the outputs.